dashboard/app/models/census/census_summary.rb

```diff
@@ -36,6 +36,7 @@ class Census::CensusSummary < ApplicationRecord
     INITIAL_IMPLEMENTATION: 0.1,
     NAIVE_BAYES: 0.2,
     SIMPLE: 0.3,
+    PRECEDENCE_V1: 1.1,
```

Comment:
What does V1 vs. 1.1 mean? :)

Reply:
It is basically arbitrary and I will happily change it if there are preferred options. We seem to have settled on using a precedence hierarchy of different data types for the summarization logic. I expect any other changes to be tweaks to this. That is why I called this V1. Similarly, I bumped the version numbering from 0.x to 1.x. I chose 1.1 so that the x in 1.x would match the 1 in V1.

Reply:
If it's just the type of heuristic we're using, then it seems we could take out the _V1 so that we just have a list of heuristic types in friendly names, and a version for each?

I'm completely naive about the use of these values though.  e.g. can we have the same version number for each of these values, or do they need to be different?  

Maybe a comment explaining how this is used in general would be useful.

Reply:
The original reason to include versioning was that we were making significant changes to the summarization logic and, in particular, the structure of the audit data. We wanted to make sure that anything processing that data would be able to know what format to expect. The version number was also useful for understanding which logic had been used to produce the current production summaries.

In practice, we never created any code that reads the audit data. It seems like it would be fine just to clear out all the old, unused version numbers at this point since nothing depends on them.